app/controllers/api/v1/invoices_controller.rb

```diff
@@ -22,6 +22,35 @@ def update
         end
       end
 
+      def show
+        invoice = Invoice.find_by(id: params[:id])
+
+        return not_found_error unless invoice
+
+        render(
+          json: ::V1::InvoiceSerializer.new(
+            invoice,
+            root_name: 'invoice',
```

Comment:
We should probably include the attached fees (or maybe add an option to include them):

```ruby
:V1::InvoiceSerializer.new(
  invoice,
  root_name: 'invoice',
  includes: %i[fees],
)
```

Reply:
Updated, I have talked with Mike also in the morning and we should send customer info as well. Thanks for the catch!